pkgs/applications/networking/cluster/openshift/default.nix

```diff
@@ -29,13 +29,15 @@ in stdenv.mkDerivation rec {
     repo = "origin";
     rev = "v${version}";
     sha256 = "06q4v2a1mm6c659ab0rzkqz6b66vx4avqfg0s9xckwhq420lzgka";
-};
+  };
+
+  goPackagePath = "github.com/openshift/origin";
 
   # go > 1.10
   # [FATAL] [14:44:02+0000] Please install Go version go or use PERMISSIVE_GO=y to bypass this check.
-  buildInputs = [ removeReferencesTo makeWrapper which go rsync go-bindata kerberos clang ];
+  buildInputs = [ which rsync go-bindata kerberos clang ];
 
-  outputs = [ "out" ];
+  outputs = [ "out" "bin" ];
```

Comment:
`bin` should become before `out`, but this is already the default so you can skip it.